tmail-backend/apps/distributed/src/main/java/com/linagora/tmail/james/app/DistributedOpenPaasAmqpFallbackModule.java

```diff
@@ -0,0 +1,33 @@
+package com.linagora.tmail.james.app;
+
+import static com.linagora.tmail.OpenPaasModule.OPENPAAS_INJECTION_KEY;
+
+import jakarta.inject.Named;
+import jakarta.inject.Singleton;
+
+import org.apache.james.backends.rabbitmq.RabbitMQConfiguration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.inject.AbstractModule;
+import com.google.inject.Provides;
+import com.linagora.tmail.AmqpUri;
+import com.linagora.tmail.configuration.OpenPaasConfiguration;
+
+public class DistributedOpenPaasAmqpFallbackModule extends AbstractModule {
```

Comment:
We need tests for both the regular behaviour (when rabbitmq uri is configured) and the fallback, for both distributed and mamory App.

In order to do that:
 - we will not create the `openpaas.properties` file
 - edit the configuration of the JamesServerExtension we are using in order to turn on the OpenPaaS extension
 - we will then need a (server) overide where we define the OpenPaaS extension configuration. It ;s ok as we already have solid tests for configuration parsing.
 
 So 4 little tests to write!